parts/global-header-container.html

```diff
@@ -0,0 +1,3 @@
+<!-- wp:group {"className":"bcgov-header-container","layout":{"type":"constrained","wideSize":"100%"}} -->
+<div class="wp-block-group bcgov-header-container"></div>
```

Comment:
are we not putting defaults in here?

Reply:
Missing template part meta has been added. Good catch.